### PR TITLE
Update UPGRADING and INSTALL

### PR DESCRIPTION
**Pull request by stefaweb**
Various mods in doc.

### CODE REVIEW

File: UPGRADING [hunk 1]
```diff
@@ -34,8 +34,7 @@ UPGRADING FROM 1.0.3 TO 1.2.0
 
 ATTENTION:
 
-Between 1.0.3 or 1.2.0 RCx version many files used by MailWatch have changed of name 
-and changed of place.
+Between 1.0.3 or 1.2.0 RCx, several files have changed their name and location.
```
Comment (endelwar) [inline UPGRADING]: Refer always to 1.2.0, not RC version which are only for testing

File: tools/Cron_jobs/INSTALL [hunk 2]
```diff
@@ -40,7 +40,6 @@ See the instructions in tools/MailScanner_rule_editor/INSTALL
 
 ATTENTION:
 
-Between 1.0.3 or 1.2.0 RCx version many files used by MailWatch have changed of name 
-and changed of place.
+Between 1.0.3 or 1.2.0 RCx, several files have changed their name and location.
```
Comment (endelwar) [inline tools/Cron_jobs/INSTALL]: Refer always to 1.2.0, not RC version which are only for testing

File: tools/Postfix_relay/INSTALL [hunk 3]
```diff
@@ -40,7 +40,6 @@ And you run it from cron every hour.
 
 ATTENTION:
 
-Between 1.0.3 or 1.2.0 RCx version many files used by MailWatch have changed of name 
-and changed of place.
+Between 1.0.3 or 1.2.0 RCx, several files have changed their name and location.
```
Comment (endelwar) [inline tools/Postfix_relay/INSTALL]: Refer always to 1.2.0, not RC version which are only for testing

File: tools/Sendmail-Exim_queue/INSTALL [hunk 4]
```diff
@@ -19,7 +19,6 @@ Note: mailwatch_sendmail_queue.php re-creates all entries on each run, so for bu
 
 ATTENTION:
 
-Between 1.0.3 or 1.2.0 RCx version many files used by MailWatch have changed of name 
-and changed of place.
+Between 1.0.3 or 1.2.0 RCx, several files have changed their name and location.
```
Comment (endelwar) [inline tools/Sendmail-Exim_queue/INSTALL]: Refer always to 1.2.0, not RC version which are only for testing

File: tools/Sendmail_relay/INSTALL [hunk 5]
```diff
@@ -21,7 +21,6 @@ By default, mailwatch_sendmail_relay.php run with 'root' user. Change user to yo
 
 ATTENTION:
 
-Between 1.0.3 or 1.2.0 RCx version many files used by MailWatch have changed of name 
-and changed of place.
+Between 1.0.3 or 1.2.0 RCx, several files have changed their name and location.
```
Comment (endelwar) [inline tools/Sendmail_relay/INSTALL]: Refer always to 1.2.0, not RC version which are only for testing

**Comment by endelwar** [comment]
These updates should go also on online docs

**Comment by stefaweb** [comment]
> These updates should go also on online docs

Will do this but later. For the moment, I prefer to track bugs.

File: UPGRADING [hunk 6]
```diff
@@ -34,8 +34,7 @@ UPGRADING FROM 1.0.3 TO 1.2.0
 
 ATTENTION:
 
-Between 1.0.3 or 1.2.0 RCx version many files used by MailWatch have changed of name 
-and changed of place.
+Several files have changed their name and location inb MailWatch 1.2.0
```
Comment (endelwar) [inline UPGRADING]: typo `inb`, also in other 4 modified files

**Comment by stefaweb** [comment]
Ready to merge.

File: README.md [hunk 7]
```diff
@@ -2,32 +2,36 @@
 
 # MailWatch for MailScanner
 
-MailWatch for MailScanner is a web-based front-end to MailScanner written in PHP, MySQL and JpGraph and is available for free under the terms of the GNU Public License.
+MailWatch for MailScanner is a web-based front-end to MailScanner written in PHP, MySQL, JpGraph and and others usefull libraries 
 
-It comes with a CustomConfig module for MailScanner which causes MailScanner to log all message data (excluding body text) to a MySQL database which is then queried by MailWatch for reporting and statistics.
+This program is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation; either version 2 of the License, or (at your option) any later version.
+
+It comes with custom PERL modules for MailScanner which causes MailScanner to log all message data (excluding body text email) to a MySQL database which is then queried by MailWatch for reporting and statistics.
```
Comment (endelwar) [inline README.md]: let's keep the keyword `CustomConfig` that is used in MailScanner docs

File: tools/Sendmail_queue/INSTALL [hunk 8]
```diff
@@ -2,24 +2,28 @@
...
 
-Note: mailwatch_sendmail_queue.php re-creates all entries on each run, so for busy sites you will probably want to change this to run every 5 minutes or greater.
+Note: mailwatch_sendmail_queue.php re-creates all entries on each run, 
+so for busy sites you will probably want to change this to run every 5 minutes or greater.
 
 ATTENTION:
 
-Between 1.0.3 or 1.2.0 RCx version many files used by MailWatch have changed of name 
-and changed of place.
+Since MailWatch 1.0.2, you no longer need to use mailwatch_sendmail_queue.php
```
Comment (endelwar) [inline tools/Sendmail_queue/INSTALL]: 1.2.0

Comment (stefaweb) [inline tools/Sendmail_queue/INSTALL]: Done

File: README.md (hunk 7 above)
Comment (stefaweb) [inline README.md]: ok

**Comment by stefaweb** [comment]
Do not merge.

I've found a problem mailwatch_sendmail_queue.php and Exim.

If we do not run it, we can no longer see the message In/out with mailq.php?queue=

I've to correct the docs.

**Comment by endelwar** [comment]
For exim queue to display like postfix is required the equivalent of `postfixmailq.php`

**Comment by stefaweb** [comment]
Should be ok to merge now.

File: README.md [hunk 9]
```diff
@@ -2,32 +2,36 @@
...
 * Drill-down onto each message to see detailed information.
-* Quarantine management allows you to release, delete or run `sa-learn` across any quarantined messages.
-* Reports with customisable filters and graphs by JpGraph
+* Quarantine management allows you to release, delete or run SpamAssasin `sa-learn` across any quarantined messages.
+* Reports with customisable filters and graphs by JpGraph.
 * Tools to view Virus Scanner status, MySQL database status and to view the MailScanner configuration files.
 * Utilities for Postfix and Sendmail to monitor and display the mail queue sizes and to record and display message relay information.
 * Multiple user levels: user, domain and admin that limit the data and features available to each.
 * XML-RPC support that allows multiple MailScanner/MailWatch installations to act as one.
+* Works with MySQL 5.5+ / MariadDB, PHP 5.3.7 to PHP 7, and have been tester on most popular Linux distributions (Debian/Umbuntu, CentOS and RedHat).
```
Comment (endelwar) [inline README.md]: typo on MariaDB and Ubuntu

File: UPGRADING [hunk 10]
```diff
@@ -19,23 +19,28 @@ UPGRADING FROM 1.0.3 TO 1.2.0
...
+
 - Run upgrade.php to update database schema. Look for errors and correct them if needed.
 
 - Start MailScanner and your MTA. Check the log in /var/log/syslog.
 
 ATTENTION:
 
-Between 1.0.3 or 1.2.0 RCx version many files used by MailWatch have changed of name 
-and changed of place.
+Several files have changed their name and location in MailWatch 1.2.0
```
Comment (endelwar) [inline UPGRADING]: this line should end with a comma (`,`) and joined with the bottom line

File: README.md (hunk 9 above)
Comment (endelwar) [inline README.md]: also `tester` is spelled wrong

File: UPGRADING (hunk 10 above)
Comment (stefaweb) [inline UPGRADING]: done

File: README.md (hunk 9 above)
Comment (stefaweb) [inline README.md]: done

**Pull request merged**